### ISSUE TITLE
Migration path from Owncloud >= 10.14?

### ISSUE THREAD

**Issue by someone-somenet-org** [nextcloud/documentation#12199]
Since Kiteworks (owncloud owner) seems to have increased their subscription prices there might be increased interest to migrate to Nextcloud.

Could you therefore update the migration page to the newest possible info there is:
https://docs.nextcloud.com/server/latest/admin_manual/maintenance/migrating_owncloud.html

**Comment by someone-somenet-org** [comment]
Thanks.

**Comment by someone-somenet-org** [comment]
Maybe I was too fast.

owncloud is currently in 10.15.0, the docs refer to 10.13.x -> NC.

Will it work with 10.15.0 too?

**Comment by criegerde** [comment]
Any updates yet? 
Related to [Ticket#9668648]

**Comment by joshtrichards** [comment]
Cc: @viktorix, @Rello

**Comment by Rello** [comment]
Hi,
I just executed my local migration with 10.15 and did not see any main issues 👍🏻

**Comment by Rello** [comment]
Hello @someone-somenet-org 
the php version does matter. oC only works with php7.4. NC30 only works with php8.x

what you can try is to stop oC; upgrade the whole Server to the newest OS/PHP and then start the occ upgrade on NC30.
Its not officially supported. But you have proper Backups in place, do you?

**Comment by criegerde** [comment]
We were able to migrate several instances from owncloud 10.15 latest » Nextcloud 25 latest » Nextcloud HUB9.
What made the trick was firstly to change the versionstring to 10.13.0.4 within the config.php and secondly to create the folder called 
`/var/www/nextcloud/apps-external`
as described [here](https://www.c-rieger.de/migration-von-owncloud-zu-nextcloud/#owncloud101x).

**Comment by susnux** [comment]
> The question is: is OC10.15 -> NC25 -> NC26 -> NC27.1 -> NC28 -> NC29 -> NC30 more supported than OC10.15 -> NC30 (and why)?

Yes this is the supported migration path, to make sure all migrations (speaking of e.g. database migrations or app data) are run correctly.

**Comment by criegerde** [comment]
I do recommend the standard/supported migration path only.
The db differs from oC to Nc - you will have troubles.

**Comment by cortho** [comment]
Last week I received an email from kiteworks saying "that ownCloud 10.14 and all versions earlier will reach end-of-life by December 31, 2025."

I'd highly appreciate direct migration support from ownCloud 10.15 to nextcloud 31.

@criegerde Thank you for your documentation which is very helpful until then.

**Comment by susnux** [comment]
https://github.com/nextcloud/documentation/pull/13026
